generated/nidaqmx/system/device.py

```diff
@@ -1069,6 +1071,17 @@ def reset_device(self):
         self._interpreter.reset_device(
             self._name)
 
+    def restore_last_ext_cal_const(self, device_name):
```

Comment:
We don't need `device_name` here, we have `self._name`

Reply:
Thanks for pointing it out, `device_name` is removed.